services/subscription.go

```diff
@@ -153,7 +156,7 @@ func TopicFiltersForRunLog(jobID string) [][]common.Hash {
 	hexJobID := common.BytesToHash([]byte(jobID))
 	jobIDZeroPadded := common.BytesToHash(common.RightPadBytes(hexutil.MustDecode("0x"+jobID), utils.EVMWordByteLen))
```

Comment:
`jobIDZeroPadded` is a little bit of a weird name for a `Hash`, because I think of zero padding strings, but not hashes (which are byte arrays).

Reply:
I agree. I was following the naming convention of the go-ethereum codebase, where as far as I can tell `common.Hash` is used to refer to anything that is 32 bytes long. For example, [common.BytesToHash](https://github.com/ethereum/go-ethereum/blob/master/common/types.go#L51) doesn't hash anything, it just truncates the number of bytes and returns them in an array. I would normally refer to that as an "EVM word." Maybe that's a better name?

Reply:
https://solidity.readthedocs.io/en/develop/abi-spec.html explains it a little more: the EVM stores variable sized values (such as `bytes`) in the higher order bits of a word, with big endian encoding.